test/library/packages/LinearAlgebra/correctness/testCholeskyComplex.chpl

```diff
@@ -0,0 +1,8 @@
+use LinearAlgebra;
```

Comment:
How about we clarify that these tests are checking an error case in the file name, e.g. `testCholeskyComplexError.{chpl,good}` and `testCholeskyRealError.{chpl,good}`